include/simdjson/generic/ondemand/value-inl.h

```diff
@@ -144,6 +144,13 @@ simdjson_really_inline simdjson_result<json_type> value::type() noexcept {
   return iter.type();
 }
 
+simdjson_really_inline simdjson_result<bool> value::scalar() noexcept {
```

Comment:
I know I'm way late to the party, but we should be careful to be consistent about boolean naming or things will get hard to use real fast. Should it be `is_scalar` or do we omit `is` in other places?